Gopkg.toml

```diff
@@ -97,6 +97,14 @@ required = [
   branch = "master"
   name = "golang.org/x/net"
 
+[[override]]
+  name = "istio.io/api"
+  version = "1.5.0"
```

Comment:
Yes, it's compile dependency. I haven't checked the Istio version on GKE yet (1.5 was a recent release). I used 1.5 because that's the release where authentication policy becomes v1beta1, which is more stable for us.